services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.utils.dateparse import parse_datetime
+from django.contrib.auth import get_user_model
+
+from db.models import Order, Ticket
+
+User = get_user_model()
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    user = User.objects.get(username=username)
+
+    order = Order.objects.create(user=user)
+
+    for ticket_data in tickets:
+        movie_session_id = ticket_data["movie_session"]
+        row = ticket_data["row"]
+        seat = ticket_data["seat"]
```

Comment:
There is no need to allocate these variables, pass values directly while creating a ticket object